openhtf/exe/triggers.py

```diff
@@ -63,9 +57,14 @@ def AutoStop(dummy_dut_id):  # pylint: disable=invalid-name
 def PromptForTestStart(message='Provide a DUT ID in order to start the test.',
                        text_input=True):
   """Make a test start trigger based on prompting the user for input."""
+  # Cache the result in case we're looping.
```

Comment:
To elaborate some on my loop test case, I'm iterating over a list of values and passing it into the test system via config value so I get a separate test run for each iteration.  I'm also using a fixed dut serial name passed in via flag through os.system() call so I don't have to enter it through the prompt.  My testers are currently prone to raising exceptions  so I like having the separate test run instances that start through my looping script.